components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -28,20 +28,39 @@ use topology::Topology;
 use manifestjson::ManifestJson;
 use bind;
 
+/// Represents a Kubernetes manifest.
 #[derive(Debug, Clone)]
 pub struct Manifest {
+    /// The service name in the Kubernetes cluster.
     pub metadata_name: String,
+    /// The service name in the Habitat ring.
     pub habitat_name: String,
+    /// The docker image.
     pub image: String,
+    /// The amount of Services that should start in the Habitat ring.
     pub count: u64,
+    /// The relationship of an app with peers in the same ring.
     pub service_topology: Topology,
+    /// The logical group of services in the Habitat ring.
     pub service_group: Option<String>,
+    /// The name of the Kubernetes secret containing the config file - user.toml - that the user
+    /// has previously created.
     pub config_secret_name: Option<String>,
+    /// The name of the Kubernetes secret that contains the ring key, which encrypts the
+    /// communication between Habitat supervisors.
     pub ring_secret_name: Option<String>,
+
+    /// Any binds, as [`Bind`] instances.
+    ///
+    /// [`Bind`]: ../bind/struct.Bind.html
```

Comment:
Have you tested that these relative links work? Just asking cause it took me a while to find out what the correct relative path was.

Reply:
I did. I hate it that you have to point to specific html like this in the source comments. At least types from the same crate should not need to be linked.